t/05-messages/03-errors.t

```diff
@@ -175,4 +175,8 @@ cmp-ok X::OutOfRange.new(
 is-run 'class { method z { $^a } }', :err{ my @lines = $^msg.lines; @lines.grep({ !/'⏏'/ && .contains: '$^a' }) }, :exitcode{.so},
 'Use placeholder variables in a method should yield a useful error message';
 
+# https://github.com/rakudo/rakudo/pull/3289
+fails-like { "a".UInt }, X::Str::Numeric,
+'UInt passes on failure given by Int';
```

Comment:
It's better for a spectest description not to rely on internal implementation. Something like _"UInt on a string produces a Failure"_, with an adjustment for my imprefect English perhaps, would be a bit more correct.